SU-SWS/ansible-playbooks
# SITES-496: Release template and version updates for 0.0.1

# READY FOR REVIEW

# Summary
- Updating the CHANGELOG and README for a 0.0.1 version release.  As you can tell from the branch name v. code committed to this branch, I was not sure whether to use 1.0 or 0.0.1.  If we're moving in the direction of semantic versioning, I am in favor of 0.0.1, but interested in your thoughts.

# Needed By (5/31)
- Best to do everything in May; I hear June will be easier.

# Urgency
- 1

# Steps to Test

1. Checkout this branch
2. Run the migration with `ansible-playbook -i inventory/sites migration-playbook.yml`.
3. Check that `drush @acsf.test.cardinald7.sws-migration-testing vget [sitename]` returns
`sws_migration_version: 0.0.1`.

# Release Template
### What
- Initial release of playbooks and roles to setup servers on ACSF and migrate sites from on premise infrastructure to Acquia Cloud Site Factory.
- Checks for PHP in the database and logs results to /tmp/sitename/.
- Attempts to change absolute and relative paths to match ACSF domains
  and new, individual site file paths.

### So what
- We have used these scripts to migrate and launch 15 sites.  Our handling of absolute and relative paths marks what we consider substantial coverage of cases where links, files, images, etc. might be broken in the migration process.

### Now what
- While this release marks a significant amount of work, there are still several outstanding PR's that we'd like to include in subsequent point releases.  They include:
https://github.com/SU-SWS/ansible-playbooks/pull/37
https://github.com/SU-SWS/ansible-playbooks/pull/36
PR 36, we'll want to be sure to merge before our next production migration, as it should allow us to differentiate a pre-launch from launch migration.

### New Requirements and Risks
- This release does include a dependency on http://github.com/su-sws/ansible-sync, in order to safeguard production sites against accidental re-migration.

### Diff
- https://github.com/SU-SWS/ansible-playbooks/compare/7e4a6d93ae26e497a574d2cc645ab8564305a352...4b9ab91831a709065a12ff4166b95e45b0aad1cc.

# Affected Projects or Products
- Sites 2.0

# Associated Issues and/or People
- JIRA tickets: https://stanfordits.atlassian.net/browse/SITES-496
- https://stanfordits.atlassian.net/browse/SITES-595
